util2/feature.txt

```diff
@@ -0,0 +1,5 @@
+feature 1 implementation
+
+feature 2 implementation
+
+feature 3 implementation
```

Comment:
Let's add a bit more to the feature 2 implementation. It should handle multiple widgets.